doc/man7/EVP_CIPHER-AES.pod

```diff
@@ -63,6 +63,12 @@ FIPS provider:
 This implementation supports the parameters described in
 L<EVP_EncryptInit(3)/PARAMETERS>.
 
+=head1 NOTES
+
+The XTS, SIV and WRAP mode implementations do not support streaming. That
```

Comment:
This is not true for XTS. It has a stream update, but it only likes full blocks.. The last update can be a partial block (Which it can then use CTS).

Reply:
It probably should have supported buffering.

Reply:
Could you please suggest how to formulate it better then? I've basically copied what we were saying in EVP_aes_128_xts() manpage.

Reply:
Something like...
````
The OpenSSL AES-SIV and AES-WRAP implementations do not support streaming. That means to obtain correct results there can be only one [EVP_EncryptUpdate(3)](http://man.he.net/man3/EVP_EncryptUpdate) or [EVP_DecryptUpdate(3)](http://man.he.net/man3/EVP_DecryptUpdate) call after the initialization of the context.

The OpenSSL AES-XTS implementation allows streaming to be performed, but each EVP_EncryptUpdate/EVP_DecryptUpdate requires each input to be a multiple of the blocksize. Only the final  EVP_EncryptUpdate/EVP_DecryptUpdate can optionally have an input that its not a multiple of the blocksize. (If it is not a multiple then CipherText Stealing is used).
````

Reply:
Thank you, I'll update it.